scrapyd/VERSION

```diff
@@ -1 +1 @@
-1.2.0
+1.2.1
```

Comment:
This is an added feature.
We only add features in new releases
which means it will wait until 1.3.
Also, we don't bump versions in master
until it's time for a new release.
This hunk should not be committed.